nemo/core/classes/exportable.py

```diff
@@ -106,6 +102,10 @@ def _export(
         my_args = locals().copy()
         my_args.pop('self')
 
+        self.eval()
```

Comment:
where can I find the definitions for `self.eval()`, `self.parameters()`, and `self.modules()` in the abstract class?

Reply:
Those are from nn.Module - Exportable is supposed to be a Module.

Reply:
Strictly speaking, no - Exportable is a mixin class, not a base. As in Python all inheritance is virtiual - yes we could probably do it to be more 'Pythonic'. Such a refactoring definitely would require thought and its own PR.

Reply:
@XuesongYang  Exportable as Mixin class is correct, there is no need to make it "more pythonic" as Mixin is not "unpythonic" in the first place. 

Mixin offers much more flexibility long term and operates properly if used correctly with clear separation of concerns between the mixin and the actual class that implements it.